pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -244,11 +247,118 @@ func (f *Fit) EventsToRegister() []framework.ClusterEventWithHint {
 		podActionType |= framework.Update
 	}
 	return []framework.ClusterEventWithHint{
-		{Event: framework.ClusterEvent{Resource: framework.Pod, ActionType: podActionType}},
-		{Event: framework.ClusterEvent{Resource: framework.Node, ActionType: framework.Add | framework.Update}},
+		{Event: framework.ClusterEvent{Resource: framework.Pod, ActionType: podActionType}, QueueingHintFn: f.isSchedulableAfterPodChange},
+		// TODO: Ideally, it's supposed to add a QueueingHintFn for verifying whether the Pod becomes schedulable after receiving
+		// events of NodeCreated and NodeUpdated. But, we may miss NodeCreated event due to preCheck. It may cause the Pod to be
+		// stuck in the scheduling queue forever.
+		// We have to wait for implementing QueueingHint until preCheck is completely removed.
+		// See: https://github.com/kubernetes/kubernetes/issues/110175
```

Comment:
```suggestion
```

We decided to have a simple QHint for now, so no longer need the comment. But, please mention 110175 issue somewhere, probably `TODO` comment in `isSchedulableAfterNodeChange`.